core/src/test/kotlin/org/stellar/anchor/sep31/Sep31HelperTest.kt

```diff
@@ -0,0 +1,56 @@
+package org.stellar.anchor.sep31
+
+import org.junit.jupiter.api.Assertions.assertFalse
+import org.junit.jupiter.api.Assertions.assertTrue
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+import org.stellar.anchor.api.exception.BadRequestException
+
+class Sep31HelperTest {
+  @Test
+  fun test_allAmountAvailable() {
+    val txn = PojoSep31Transaction()
+    txn.amountIn = "100"
+    assertFalse(Sep31Helper.allAmountAvailable(txn))
+    txn.amountInAsset = "USDC"
+    assertFalse(Sep31Helper.allAmountAvailable(txn))
+    txn.amountOut = "500"
+    assertFalse(Sep31Helper.allAmountAvailable(txn))
+    txn.amountOutAsset = "BRL"
+    assertFalse(Sep31Helper.allAmountAvailable(txn))
+    txn.amountFee = "5"
+    assertFalse(Sep31Helper.allAmountAvailable(txn))
+    txn.amountFeeAsset = "USDC"
+    assertTrue(Sep31Helper.allAmountAvailable(txn))
+  }
+
+  @Test
+  fun test_validateStatus() {
+    val txn = PojoSep31Transaction()
+    txn.status = "pending_stellar"
+    Sep31Helper.validateStatus(txn)
+    txn.status = "pending_sender"
+    Sep31Helper.validateStatus(txn)
+    txn.status = "pending_customer_info_update"
+    Sep31Helper.validateStatus(txn)
+    txn.status = "pending_transaction_info_update"
+    Sep31Helper.validateStatus(txn)
+    txn.status = "pending_receiver"
+    Sep31Helper.validateStatus(txn)
+    txn.status = "pending_sender"
+    Sep31Helper.validateStatus(txn)
+    txn.status = "completed"
+    Sep31Helper.validateStatus(txn)
+    txn.status = "error"
+    Sep31Helper.validateStatus(txn)
```

Comment:
I was trying to use the test code to lock the list of the `SepHelper.sep31Statuses` to prevent accidental modification of the list. 

I had revised the code to use `@ValueSource` to avoid the repetition.